src/Config.php

```diff
@@ -308,16 +308,22 @@ protected function parseGeneral()
                 // as a regex-like string, and we switched to an array. If we find the old style, fall back to the defaults.
                 unset($general['accept_file_types']);
             }
-            // accept uppercase and lowercase file extensions
-            $general['accept_file_types'] = array_unique(
-                array_merge(
-                    $general['accept_file_types'],
-                    array_map(function ($extension) {
-                        return strtolower($extension);
-                    }, $general['accept_file_types']),
-                    array_map(function ($extension) {
-                        return strtoupper($extension);
-                    }, $general['accept_file_types'])
+
+            // Just.. Say no to these.
+            $general['accept_file_types'] = array_diff($general['accept_file_types'], ['sh', 'asp', 'cgi', 'php', 'php3', 'ph3', 'php4', 'ph4', 'php5', 'ph5' ,'phtm', 'phtml']);
+
+            // accept uppercase and lowercase file extensions.
+            $general['accept_file_types'] = array_values(
```

Comment:
use collection (bolt has one)

Reply:
@JarJak You could have done this when you introduced this bit of code in #7751. I'm fine with it the way it is now.

Reply:
Yeah, right.. I just didn't know Bolt have it's own Collection implementation when I have done this though.

Reply:
Ah, ok.. It's been taken care of now. :-)